ingest/ingest_pipeline.py

```diff
@@ -153,7 +155,8 @@ def get_mongo_db(self):
         user = os.environ['MONGODB_USERNAME']
         password = os.environ['MONGODB_PASSWORD']
         db_name = os.environ['DATABASE_NAME']
```

Comment:
Will `MongoConnection()` replace `def get_mongo_db()`?  I assume so.

Could you add a comment in the latter if it'll be replaced, or if not what makes its use case different from `MongoConnection()`?

Reply:
40d0d5510a7af0b0b1dbc942b38444a986ff058f

Reply:
?